azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/compute/AzureComputeServiceContextLiveTest.java

```diff
@@ -236,24 +245,7 @@ public void testLinuxNode() throws RunNodesException {
       final TemplateBuilder templateBuilder = view.getComputeService().templateBuilder();
       templateBuilder.osFamily(OsFamily.UBUNTU);
       templateBuilder.osVersionMatches("14.04");
-      templateBuilder.hardwareId("Standard_A0");
-      templateBuilder.locationId("westus");
-      final Template template = templateBuilder.build();
-
-      try {
-         Set<? extends NodeMetadata> nodes = view.getComputeService().createNodesInGroup(groupName, 1, template);
-         assertThat(nodes).hasSize(1);
-      } finally {
-         view.getComputeService().destroyNodesMatching(inGroup(groupName));
-      }
-   }
-
-   @Test(dependsOnMethods = "testLinuxNode")
-   public void testWindowsNode() throws RunNodesException {
-      final String groupName = this.azureGroup;
-      final TemplateBuilder templateBuilder = view.getComputeService().templateBuilder();
-      templateBuilder.imageId("global/MicrosoftWindowsServer/WindowsServer/Windows-Server-Technical-Preview");
-      templateBuilder.hardwareId("Standard_A0");
+      templateBuilder.hardwareId("Standard_A5");
       templateBuilder.locationId("westus");
       final Template template = templateBuilder.build();
 
```

Comment:
Consider removing this class. Tests here should already be covered by the `AzureComputeServiceLiveTest`.